src/libs/getPolicyDomainName.ts

```diff
@@ -0,0 +1,3 @@
+export default function getPolicyDomainName(policyID: string): string {
+    return `expensify-policy${policyID}.exfy`;
```

Comment:
@narefyev91 actually relaized, this has to be lowercase, can you make sure the returned string is made lowercase?

Reply:
@mountiny you mean policyID value?

Reply:
updated

Reply:
Yeah the entire thing but the policyID in there would be uppercase

Reply:
yeah i found already existed util function for that